From 14d23644e272f0cfacfe526c70201d384454bd95 Mon Sep 17 00:00:00 2001
From: Ye Li <ye.li@nxp.com>
Date: Mon, 3 Sep 2018 00:44:31 -0700
Subject: [PATCH] MLK-19433-1 imx: Add macro for chip rev 2.1

Add common CHIP_REV_2_1 for chip revision 2.1

Signed-off-by: Ye Li <ye.li@nxp.com>
(cherry picked from commit f7fc83ffb0f204d9f6ec6c77c08d23869d9ecde4)
(cherry picked from commit b3552a40e825094635b3a152fa9f4466fc311925)
---
 arch/arm/include/asm/arch-imx/cpu.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/include/asm/arch-imx/cpu.h b/arch/arm/include/asm/arch-imx/cpu.h
index 3bc08e4ee6..dd392a1233 100644
--- a/arch/arm/include/asm/arch-imx/cpu.h
+++ b/arch/arm/include/asm/arch-imx/cpu.h
@@ -43,6 +43,7 @@
 #define CHIP_REV_1_2            0x12
 #define CHIP_REV_1_5            0x15
 #define CHIP_REV_2_0            0x20
+#define CHIP_REV_2_1            0x21
 #define CHIP_REV_2_5            0x25
 #define CHIP_REV_3_0            0x30
 
